src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,43 @@
+package duke.task;
+
+import java.io.Serializable;
+
+public class Task implements Serializable {
+    //Description of Task
+    protected String description;
+    //Flag for whether a task is done, default false
+    protected boolean isDone;
+
+    /**
+     * Constructor for Task.
+     *
+     * @param description String description of Task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done duke.task with X
```

Comment:
This code should be self explanatory enough to be able to omit this comment? 🤔

Reply:
haha true